nixos/modules/services/x11/urxvtd.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+# maintainer: siddharthist
+
+with lib;
+
+let
+  cfg = config.services.urxvtd;
+in {
+
+  options.services.urxvtd.enable = mkOption {
+    type = types.bool;
+    default = false;
+    example = true;
+    description = ''
+      Enable urxvtd, the urxvt terminal daemon. To use urxvtd, first
+      "export RXVT_SOCKET=/run/user/$(id -u)/urxvtd-socket" then run
+      "urxvtc".
```

Comment:
works so but a currently I have to manually start urxvtd: systemctl --user start urxvtd
If this indented, it should be documented.